pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,16 +44,16 @@ const (
 	SharesPerCPU  = 1024
 	MilliCPUToCPU = 1000
 
-	// 100000 is equivalent to 100usec
-	QuotaPeriod    = 100000
-	MinQuotaPeriod = 1000
+	// 100000 is equivalent to 100ms
+	QuotaPeriodMicroSeconds    = 100000
+	MinQuotaPeriodMicroSeconds = 1000
```

Comment:
Do we need to rename consts?

Reply:
Good question. I think we don't, I will settle on adding a comment for these.